src/Components/UI/Navigation/Navigation.css

```diff
@@ -0,0 +1,75 @@
+.navigation{
+    border-bottom: 3px solid var(--nav-color);
+    background-color: rgba(0,0,0,0.5);
+    backdrop-filter: blur(5px);
+}
+
+.burger{
+    display: inline-block;
+    cursor: pointer;
+}
+
+.nav-search {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    width: 100%;
+    height: 100%;
+}
+
+.nav-search input {
+    max-width: 35rem;
+}
+
+.navigation .icons {
+    align-items: center;
+    min-width: 13rem;
+}
+
+.nav-search input {
+    max-width: 35rem;
+}
+
+.nav-search button {
+    border: none;
+    background: none;
+    padding: 0;
+    margin: 0;
```

Comment:
Do add reset CSS I see a lot of places you are doing this.

Reply:
Noted. Will fix it in next iteration as some things are not looking as expected.